.github/workflows/publish.yaml

```diff
@@ -0,0 +1,68 @@
+name: Publish Container Images
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  # we need to regularly rebuild the image to get the latest cnspec image
+  schedule:
```

Comment:
Can we chat about this approach during standup. I have concerns about what this means for versions

Reply:
As discussed, I will try out @chris-rock suggestion.

Reply:
Ok, next iteration with another approach. No if, no wrapper, just not using the docker action.